app/assets/javascripts/custom.js

```diff
@@ -32,7 +32,7 @@ $(document).ready(function() {
       url: "/api/v1/games/" + gameId,
       data: gameParams,
       success: function () {
-        $("p").html("Game was successfully updated. Return to all games to see changes.");
+        window.location = "/" + gameParams.game.name;
```

Comment:
Suggestion: could add some error handling here. E.g. if the request doesn't go through, you could display a message to the user.